src/components/Header.js

```diff
@@ -2,16 +2,12 @@ import React, { Component } from "react";
 import {
   Collapse,
   Container,
-  DropdownItem,
-  DropdownMenu,
-  DropdownToggle,
   Nav,
   Navbar,
   NavbarBrand,
   NavbarToggler,
   NavItem,
-  NavLink,
-  UncontrolledDropdown,
+  NavLink
```

Comment:
Having this come from Reactstrap is going to cause some internal navigation issues. You don't want hrefs for your internal links.